specification/deviceregistry/DeviceRegistry.Management/tspconfig.yaml

```diff
@@ -17,6 +17,10 @@ options:
     namespace: "com.azure.resourcemanager.deviceregistry"
     service-name: "Device Registry"
     examples-directory: "examples"
+  "@azure-tools/typespec-csharp":
+    namespace: "Azure.ResourceManager.DeviceRegistry"
+    service-name: "Device Registry"
```

Comment:
What are service-name and examples-directory for? I think we don't use them?

Reply:
you are right, csharp is not consuming them yet, should be removed.